cloud/endagaweb/templates/dashboard/layout.html

```diff
@@ -21,15 +21,15 @@
       {% endblock %}
     </title>
 
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.0.3/css/bootstrap.min.css" type="text/css"/>
-    <link href="/static/css/dashboard.css" rel="stylesheet">
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.0.3/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.0.3/css/bootstrap.min.css" type="text/css" />
+    <link href="/static/css/dashboard.css" rel="stylesheet" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.0.3/css/font-awesome.min.css" />
 
     <!-- Custom styles for this template -->
     {% block pagestyle %}
     {% endblock %}
 
-    {% if user_profile.user.is_staff %}
+    {% if user_profile.user.is_superuser%}
```

Comment:
As we have added User Management feature in which we have differentiated Network/Cloud admin on the basis of the staff and superuser, i.e why we have updated the checks, in demo this can be explained in detail.

Fixed trailing space